src/controllers/login.js

```diff
@@ -0,0 +1,57 @@
+const getUser = require('./../database/get_login');
```

Comment:
1 file filled with commented code, and what are you working on? Events or login? One should not be here.